test/models/site_updates/new_exercise_update_test.rb

```diff
@@ -38,7 +38,8 @@ class SiteUpdates::NewExerciseUpdateTest < ActiveSupport::TestCase
       [author].map do |maker|
         {
           handle: maker.handle,
-          avatar_url: maker.avatar_url
+          avatar_url: maker.avatar_url,
+          flair: maker&.flair
```

Comment:
This also applies to the other occurences in this file

```suggestion
          flair: maker.flair
```